src/main.c

```diff
@@ -115,11 +115,22 @@ void main_initthr(void *unused)
 	proc_fileSet(2, 3, &oid, 0, 0);
 
 	if (last != NULL)
```

Comment:
I suggest using brackets for if (last != NULL)

Reply:
done